pkg/gadgets/seccomp/gadget.go

```diff
@@ -135,6 +137,48 @@ func (t *Trace) Operation(trace *gadgetv1alpha1.Trace,
 	}
 }
 
+type pubSubKey string
+
+func genPubSubKey(namespace, name string) pubSubKey {
+	return pubSubKey(fmt.Sprintf("gadget/seccomp/%s/%s", namespace, name))
+}
+
+func (t *Trace) containerEvent(trace *gadgetv1alpha1.Trace, event pubsub.PubSubEvent) {
+	if event.Type != pubsub.EVENT_TYPE_REMOVE_CONTAINER {
+		return
+	}
+	if trace.Spec.OutputMode != "ExternalResource" {
+		return
+	}
+	if event.Container.Mntns == 0 {
+		log.Errorf("Container has unknown mntns")
+		return
+	}
+
+	b := traceSingleton.tracer.Peek(event.Container.Mntns)
+	traceSingleton.tracer.Delete(event.Container.Mntns)
+
+	var r *seccompprofilev1alpha1.SeccompProfile
+	r = syscallArrToSeccompPolicy(trace.ObjectMeta.Namespace, trace.ObjectMeta.Name, b)
+
+	// Copy labels from the trace into the SeccompProfile. This will allow
+	// the CLI to add a label on the trace and gather its output
+	if trace.ObjectMeta.Labels != nil {
+		for key, value := range trace.ObjectMeta.Labels {
+			r.ObjectMeta.Labels[key] = value
+		}
+	}
+	pod := fmt.Sprintf("%s/%s", event.Container.Namespace, event.Container.Name)
```

Comment:
It says pod but it's actually taking the name from a container. Will it work with pods with multiple containers?